src/Uno.UI.RemoteControl.Server.Processors/HotReload/MetadataUpdates/WatchHotReloadService.cs

```diff
@@ -20,7 +20,7 @@ public class WatchHotReloadService
 		private Action _endSession;
 		private object _targetInstance;
 
-		public readonly struct Update
+		public readonly struct Update : IEquatable<Update>
```

Comment:
Overall, I find that the maintainability of this kind of changes pretty low. This particular change would benefit from being a record.

For the others, where records are a breaking change, would you know of an existing generator that would do that for structs (we had [one similar in Uno.CodeGen](https://github.com/unoplatform/Uno.CodeGen/blob/master/src/Uno.CodeGen/EqualityGenerator.cs))? Otherwise, I don't necessarily like regions but having one that explicitly says that the equality related members should be updated on any change would be interesting.

Reply:
I'll update `Update` struct to be a `record struct`.

For the others, I don't know of an existing generator, I'll have a region and a comment to note they should be updated for new fields.